features/networking/sdn.feature

```diff
@@ -392,3 +392,30 @@ Feature: SDN related networking scenarios
       | since         | 30s                            |
     And the output should contain:
       | Not applying unsafe change: invalid configuration |
+
+
+  # @author zzhao@redhat.com
+  # @case_id OCP-33206
+  @admin
+  Scenario: sdn metrics should be using https scheme
+    Given I switch to cluster admin pseudo user
+    When I use the "openshift-sdn" project
+    Then the expression should be true> service_monitor('monitor-sdn').service_monitor_endpoints_spec.first.scheme == 'https'
```

Comment:
@zhaozhanqi @anuragthehatter pointed out in #1461 that the endpoint names are different for OCP < 4.4.  So if you want this testcase to work for all version of OCP, we need to change the logic a bit.

Reply:
@pruan-rht This is a specific 4.6 feature so no need to implement that logic here. Right @zhaozhanqi ?

Reply:
Then perhaps add a filter at the beginning to make sure the ocp version is >= 4.6

Reply:
yes, this case need version >=4.6.  I added the step >=4.6
Actually. this case cannot be selected for <4.6 by case filter.